src/player.js

```diff
@@ -2074,7 +2074,11 @@ export default class Player extends FakeEventTarget {
         const videoElement = this.getVideoElement();
         return videoElement ? Array.from(videoElement.textTracks).findIndex(track => (track ? track.language === textTrack.language : false)) : -1;
       };
-      this._getTextTracks().forEach(track => (track.index = getNativeLanguageTrackIndex(track)));
+      this._getTextTracks().forEach(track => {
+        if (!track.emptyLanguage) {
```

Comment:
Why do we need this new property called emptyLanguage and the overhead of handing it (all the changes in the TextTrack class). Can't we just check here "if (track.language) {" ?

Reply:
Oh, you want it targeted only for the NativeAdapter?

Reply:
But I guess this code is correct always... Even if it doesn't happen in other adapters. If there is no language then don't try to look it up.

Reply:
no cause it's not empty

Reply:
we set value on native adapter for CEA708